en/docs/samples/role-based-adaptive-auth.md

```diff
@@ -0,0 +1,96 @@
+# Configure Role-Based Adaptive Authentication
+
+This page guides you through configuring role-based adaptive authentication for a sample web application using a sample hardware key authenticators. 
+
+----
+
+If you have have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../guides/configure-adaptive-auth" target="_blank" rel="nofollow noopener">I have my own application</a>
+
+----
+
+## Scenario
+
+Consider a scenario where you want a user who belongs to the 'admin' role to perform an additional level of authentication while any other user can just provide their credentials (basic authentication) to access a resource.
```

Comment:
Shall we rephrase this to be, 
authentication, while other users can access a resource by providing their credentials (basic authentication) alone.